bitrixSM/internal/transport/handlers/createTaskr.go

```diff
@@ -5,42 +5,52 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"strconv"
+	"strings"
 
 	"github.com/gin-gonic/gin"
 )
 
 type createTaskParams struct {
-	Machineid    int64  `json:"machineid"`
-	Shiftid      int64  `json:"shiftid" `
-	Frequency    string `json:"frequency"`
-	Taskpriority string `json:"taskpriority"`
-	Description  string `json:"description"`
-	Createdby    int64  `json:"userid"`
+	Machineid    int64
+	Shiftid      int64
+	Frequency    string
+	Taskpriority string
+	Description  string
+	Createdby    int64
 }
 
 func CreateTask(c *gin.Context) error {
+	rawUserID, exists := c.Get("userID")
+	if !exists {
+		return fmt.Errorf("userID not found in context")
+	}
+	message := c.GetString("message")
```

Comment:
For now for me it is not clear why do we need to keep message in context, let's discuss it

Reply:
this file contains a lot of magic strings, ask to comment it on the meet with me when you will be ready